README.md

```diff
@@ -255,14 +255,20 @@ dependency as follows, replacing `{{artifact-id}}` with the value from the "Arti
 | [OTLP Common](./exporters/otlp/common)              | Shared OTLP components (internal)                                                   | `opentelemetry-exporter-otlp-common`  | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
-| [Jaeger gRPC Exporter](./exporters/jaeger)          | Jaeger gRPC trace exporter                                                          | `opentelemetry-exporter-jaeger`       | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
-| [Jaeger Thrift Exporter](./exporters/jaeger-thrift) | Jaeger thrift trace exporter                                                        | `opentelemetry-exporter-jaeger-thift` | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
+| [Jaeger gRPC Exporter](./exporters/jaeger)          | Jaeger gRPC trace exporter (deprecated [1])                                         | `opentelemetry-exporter-jaeger`       | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
+| [Jaeger Thrift Exporter](./exporters/jaeger-thrift) | Jaeger thrift trace exporter (deprecated [1])                                       | `opentelemetry-exporter-jaeger-thift` | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
 | [Logging Exporter](./exporters/logging)             | Logging exporters, includings metrics, traces, and logs                             | `opentelemetry-exporter-logging`      | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
 | [OTLP Logging Exporter](./exporters/logging-otlp)   | Logging exporters in OTLP protobuf JSON format, including metrics, traces, and logs | `opentelemetry-exporter-logging-otlp` | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
 | [Zipkin Exporter](./exporters/zipkin)               | Zipkin trace exporter                                                               | `opentelemetry-exporter-zipkin`       | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
 | [Prometheus Exporter](./exporters/prometheus)       | Prometheus metric exporter                                                          | `opentelemetry-exporter-prometheus`   | <!--VERSION_UNSTABLE-->1.22.0-alpha<!--/VERSION_UNSTABLE--> |
 | [Exporter Common](./exporters/common)               | Shared exporter components (internal)                                               | `opentelemetry-exporter-common`       | <!--VERSION_STABLE-->1.22.0<!--/VERSION_STABLE-->           |
 
+**[1]**: Jaeger now
+has [native support for OTLP](https://opentelemetry.io/blog/2022/jaeger-native-otlp/) and jaeger
+exporters are now deprecated. `opentelemetry-exporter-jaeger-thrift` will continue to be published
+until 1.27.0 (July 2023). After 1.27.0, it will continue to receive patches for security
+vulnerabilities, and `io.opentelemetry:opentelemetry-bom` will reference the last published version.
```

Comment:
Shouldn't we instead plan for removal after 1.27.0? When a vuln is discovered in a dependency of the jaeger clients, it could prove difficult to continue patching in the otel exporters. Maybe I misunderstand the intent here, but the way it's currently written makes it sound like there will be indefinite support for security fixes in the jaeger exporters.

Reply:
> Shouldn't we instead plan for removal after 1.27.0?

In the past when we've stopped publishing artifacts we've said we'll continue to patch bugs / security fixes. Bug fixes make sense for problems on our side. But as you note, indefinite security fixes are problematic, especially when an artifact depends no a deprecated library. The `opentelemetry-exporter-jaeger` exporter doesn't depend the deprecated jaeger client libraries, so would be straight forward to provide security fixes for. `opentelemetry-exporter-jaeger-thrift` does depend on the deprecated jaeger client libraries.

Maybe we just say we'll patch bugs?

Reply:
See my latest comment [here](https://github.com/open-telemetry/opentelemetry-java/pull/5190#issuecomment-1611959161).